data/repos.yml

```diff
@@ -307,6 +307,12 @@
   team: "#tech-content-interactions-on-platform-govuk"
   production_hosted_on: eks
 
+- repo_name: fake-ckan_harvest_source
```

Comment:
This repo should probably be archived too as it's superseded by clan-mock-harvest-sources repo

Reply:
I see you've now archived it. Thanks!